build-tools/build.sh

```diff
@@ -0,0 +1,248 @@
+#!/bin/bash
+script_start_time=$SECONDS
+# ------------
+# ECHO LICENSE
+# ------------
+echo "::group::License"
```

Comment:
What do the colons mean in these `echo` calls?

Reply:
It’s “GitHub Workflow Commands”. See https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions